pyproject.toml

```diff
@@ -46,7 +61,10 @@ maintainers = [
 #     https://scipy.github.io/devdocs/dev/core-dev/index.html#version-ranges-for-numpy-and-other-dependencies
 requires-python = ">=3.10"
 dependencies = [
-    "numpy>=1.23.5",
+    # free-threaded CPython 3.13 support requires more
+    # recent NumPy release at runtime
+    "numpy>=2.1.3,<2.5; python_version=='3.13t'",
+    "numpy>=1.23.5,<2.5; python_version!='3.13t'",
```

Comment:
This also doesn't work of course, probably just do the usual. At least for this runtime case, I believe could catch the problem of lowest NumPy req in `__init__.py` for free-threaded if we wanted though?